config/jobs/kubernetes/minikube/minikube.yaml

```diff
@@ -1,64 +0,0 @@
-presets:
-- labels:
-    preset-minikube-e2e-creds: "true"
-  env:
-  - name: GCS_CREDS
-    value: /root/.config/gcloud/k8s-minikube-build-gcs.json
-  volumes:
-  - name: k8s-minikube-build-gcs
-    secret:
-      secretName: k8s-minikube-build-gcs
-  volumeMounts:
-  - name: k8s-minikube-build-gcs
-    mountPath: /root/.config/gcloud
-    readOnly: true
-
-presubmits:
-  kubernetes/minikube:
-  - name: pull-minikube-build
```

Comment:
could you please do not remove this build job(and just replace the prow-test image), we still need this job for every pr.